transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Documents/API.java

```diff
@@ -2,12 +2,25 @@
 
 import org.springframework.data.mongodb.core.mapping.Document;
 
+@Document
+public class API {
```

Comment:
I would probably add a quick overview of what this class represents and the purpose of us having it (i.e, why/how we are using it)